### ISSUE TITLE
Version missing from release exe

### ISSUE THREAD

**Issue by cjbarth** [TheJoeFin/Text-Grab#175]
This is a minor issue, but it would help in knowing if I'm launching the latest version. When looking at properties of the executable, the details tab shows File version and Product version both as 0.0.0.0. Could the real version number be put in one of these files?

**Comment by TheJoeFin** [comment]
Agreed, this should be tracked. I wish there was an easy way to keep the version of the packaging project for the .MSIX and the version of the .EXE in sync.

**Comment by cjbarth** [comment]
I know there is a way, but I don't know your build process to offer suggestions. Perhaps [this](https://stackoverflow.com/questions/9312221/msi-version-numbers) will help.

**Comment by gitname** [comment]
I want to clarify that I think this issue is about the EXE's filename and metadata, not something within the _running_ program, itself.

#### Regarding the filename: 

As of version `2.7.0`, the `.exe` file is named, `Text-Grab.exe`, as opposed to, for example, `Text-Grab-v7.2.0.exe`.

![image](https://user-images.githubusercontent.com/7143133/187099148-e17a285f-38e7-4486-bb03-f401e669cf15.png)

#### Regarding the file's metadata:

As of that same version, as @cjbarth said, the metadata shows both a file version of `0.0.0.0` and a product version of `0.0.0.0`.

![image](https://user-images.githubusercontent.com/7143133/187099287-8e27bb8b-52e7-4d8b-b128-56c1b1752926.png)

---

When I run the program, however, I do see the version number, `v7.2.0`, at the top left of the "Settings" window.

![image](https://user-images.githubusercontent.com/7143133/187099078-d44a7960-9d67-495a-998e-714883754de8.png)

---

Awesome tool, by the way! 👍 👍 👍

**Comment by Korb** [comment]
[Sysinternals Sigcheck](https://learn.microsoft.com/en-gb/sysinternals/downloads/sigcheck) about [4.6.0 (2024-11-12)](https://github.com/TheJoeFin/Text-Grab/releases/tag/v4.6.0):

> Text-Grab.exe:
> Verified: Unsigned
> Link date: 2:11 17.09.2024
> Publisher: n/a
> Company: n/a
> Description:
> Product: n/a
> Prod version: 0.0.0.0
> File version: 0.0.0.0
> MachineType: 64-bit
> Binary Version: 0.0.0.0
> Original Name: Text-Grab.dll
> Internal Name: Text-Grab.dll
> Copyright:
> Comments: n/a
> Entropy: 6.857
> MD5: FE50108142F3B7A6ADCA3C6338666CB4
> SHA1: 04221D88FD2B7CF7531986A4F5046B54E1BADB61
> PESHA1: 9EBDF4A9E4ACBF835DB05630402BCCF680D5F01A
> PE256: 99BC95E91B7D88BE2465E5A66A74C03D8DE01E0F4092D7E92F9ED3F27A5AA131
> SHA256: 4C63E9CA0BAD9481BB0F5DAA6BFD362DAF0EE8F4106D81A9B25797A654126B50
> IMP: 4B1892CE4FBCFCF064C6F69D693FC6A5
> VT detection: 1/75
> VT link: virustotal.com/gui/file/4c63e9ca0bad9481bb0f5daa6bfd362daf0ee8f4106d81a9b25797a654126b50/detection
